tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/projects/LauncherApplication.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.central.test.ui.reddeer.projects;
+
+import org.jboss.tools.central.reddeer.projects.ArchetypeProject;
+
+/**
+ *
+ * @author zcervink
+ *
+ */
+
+public class LauncherApplication extends ArchetypeProject {
```

Comment:
@zcervink I have not found, where is this class used - if it is not used, remove it. Thank you

Reply:
deleted.